modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/OAuthOperationMessageProcessor.java

```diff
@@ -77,7 +76,7 @@ public OAuthOperationMessageProcessor(ExtensionModel extensionModel,
   @Override
   protected Mono<CoreEvent> doProcess(CoreEvent event, ExecutionContextAdapter<OperationModel> operationContext) {
     return super.doProcess(event, operationContext)
-        .onErrorResume(AccessTokenExpiredException.class, e -> {
+        .onErrorResume(Exception.class, e -> {
```

Comment:
I would add a test for this fix, where the AccessTokenExpiredException was the cause of another exception

Reply:
Added in the integration tests.